modules/restarting-the-central-db-container.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * configuration/add-trusted-ca.adoc
+// * configuration/configure-endpoints.adoc
+// * configuration/add-custom-certificates.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="restarting-the-central-db-container_{context}"]
+= Restarting the Central DB container
+
+[role="_abstract"]
+You can restart the Central DB container by killing or by deleting the Central DB pod.
+
+.Procedure
+
+* To kill the Central DB pod, run the following command:
++
+[source,terminal]
+----
+$ oc -n stackrox exec deploy/central-db -c central-db -- kill 1
+----
++
+[IMPORTANT]
+====
+Wait until {ocp} propagates your changes and restarts the Central DB container.
+====
+
```

Comment:
```suggestion
```

I think this can be removed, deleting the pod is better. This just adds complexity.